docs/04.guides/04.cookbooks/31.Externalize_Strings/page.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Externalize strings
+id: Externalizing_Strings
+---
+## Externalize strings ##
+
+Externalize strings from generated class files to separate files. This method is used to reduce the memory of the static contents for templates. We explain this method with a simple example below:
+
+**Example:**
+
+```lucee
+//index.cfm page 
+
+<cfset year = 1960>
+<html>
+	<body>
+		<h1>....</h1>
+		.......
+		.......
+		It was popular in the <cfoutput> #year# </cfoutput>
+		.......
+		<b>....</b>
+	</body>
+</html>
+```
+
+1. Here Index.cfm file contains lot of strings(static contents) inside, but there is no functionality. Just give a cfoutput with year, In year is already declared by using <cfset> in top of the Index.cfm page.  
+
+2. Execute that cfm page in browser. A class file is created in ``webapps\ROOT\WEB-INF\lucee\cfclasses\`` directrory while execute a cfm file. The run time compiler compiles that file to load the java bytecode and execute it. 
```

Comment:
***ToChange***
2. Execute the CFMpage in a browser. A class file is created in the ``webapps\ROOT\WEB-INF\lucee\cfclasses\`` directory while the CFM file is executed. The run time compiler compiles that file to load the java bytecode and execute it.